gooddata-java/src/test/java/com/gooddata/sdk/service/project/ProjectServiceAT.java

```diff
@@ -150,6 +163,20 @@ public void getUserInProject() {
         assertThat(user, notNullValue());
     }
 
+    @Test(groups = {"project", "isolated_domain"}, dependsOnMethods = "disableUserInProject")
+    public void removeAccountFromProject() {
+        final User user = gd.getProjectService().getUser(project, account1);
+        assertThat(user, notNullValue());
```

Comment:
not needed - the getUser should always return non-null value (or throw exception if not found)

Reply:
ok, will remove assert